dspace-oai/src/main/java/org/dspace/xoai/util/DateUtils.java

```diff
@@ -29,10 +29,13 @@ public static String format(Date date)
     {
         return format(date, true);
     }
+
     public static String format(Date date, boolean init)
     {
-    	SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.'000Z'");
-    	if (!init) sdf = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss.'999Z'");
+        SimpleDateFormat sdf = new SimpleDateFormat(
+                "yyyy-MM-dd'T'HH:mm:ss.'000Z'");
+        if (!init)
```

Comment:
If we add a line break here, we should use curly brackets as well. Everything else is error-prone.